llvm/test/CodeGen/NVPTX/truncate_zext.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -march=nvptx64 < %s | FileCheck %s
+
```

Comment:
+1 to that. The test should have some code which has to choose between free ops vs an alternative which would be used otherwise (e.g. `i32 trunc(add(i64,i64))` vs `i32 add(trunc(i64), trunc(i64))`)